apps/backend/.env.docker

```diff
@@ -0,0 +1,33 @@
+STACK_BASE_URL=http://localhost:8102
+STACK_SERVER_SECRET=23-wuNpik0gIW4mruTz25rbIvhuuvZFrLOLtL7J4tyo
+
+# Postgres connection strings
+STACK_DATABASE_CONNECTION_STRING=postgres://stack:stack123@postgres:5432/stack
+STACK_DIRECT_DATABASE_CONNECTION_STRING=postgres://stack:stack123@postgres:5432/stack
+
+# optionally skip migrations on container startup
+STACK_SKIP_MIGRATIONS="false"
+
+# run Prisma seed script on container startup
+STACK_RUN_SEED_SCRIPT="true"
+
+# Self-host seed script options
+# Optionally create a default admin user and disable signups to the platform when the seed script is run.
+# This can be useful for self hosted deployments where you want to restrict access to the Stack "internal" project.
+# If not specified, no user will be created and you will have to create the first user manually by signing up.
+# STACK_DEFAULT_DASHBOARD_USER_EMAIL=admin@example.com
+# STACK_DEFAULT_DASHBOARD_USER_PASSWORD=admin123
+# STACK_DEFAULT_DASHBOARD_USER_DISPLAY_NAME=Admin
```

Comment:
This feels really dangerous, it means that with the default settings everyone can login to other people's admin accounts because the password is always the same. People don't need admin access to use Stack Auth, I would just either disable this by default or leave it away entirely

Reply:
Let's do that. The seed script runs every time we launch the Docker container, right? We can just make it overwrite the trusted domain with an environment variable so it needs no config at build time and can still be configured at runtime